sources/lib/DatabaseDataCollector.php

```diff
@@ -0,0 +1,155 @@
+<?php
+/*
+ * This file is part of the PommProject/PommBundle package.
+ *
+ * (c) 2018 Grégoire HUBERT <hubert.greg@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace PommProject\PommBundle;
+
+use PommProject\Foundation\Exception\SqlException;
+use PommProject\Foundation\Listener\Listener;
+use PommProject\Foundation\Session\Session;
+
+use Symfony\Component\Stopwatch\Stopwatch;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\HttpKernel\DataCollector\DataCollector;
+
+/**
+ * Data collector for the database profiler.
+ *
+ * @package PommBundle
+ * @copyright 2018 Grégoire HUBERT
+ * @author Jérôme MACIAS
+ * @author Grégoire HUBERT
+ * @license X11 {@link http://opensource.org/licenses/mit-license.php}
+ * @see DataCollector
+ */
+class DatabaseDataCollector extends DataCollector
+{
+    /** @var Stopwatch */
+    private $stopwatch;
+
+    public function __construct($unused = null, Stopwatch $stopwatch = null)
+    {
+        if ($unused !== null) {
+            trigger_error("The parameter Pomm has been deleted for to delete the high dependency.", E_USER_DEPRECATED);
```

Comment:
Maybe rephrase it to `The Pomm parameter has been deleted in order to remove the higher dependency` or somethin ?